### ISSUE TITLE
Plugin Details: “Install” cell should be all one element saying “Install version x”

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12913]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > Plugins.
1. Swipe right to find the search field.
1. Search for "Hello Dolly" and double tap to open the plugin detail page.
1. Swipe right to read through the elements on the page.

**Result:** [16s](https://cloudup.com/chaox-IUr5e)

* Title cell should be all one element including the author name.
* “Install” cell should be all one element saying “Install version x”.
* Description cell should be all one element.

<img src="https://user-images.githubusercontent.com/1119271/68520507-8c725400-0255-11ea-8a3e-01ae68a366ca.png" width="270" alt="plugin-list-install-button-highlighted-voiceover">
<sup>Tested with WP Internal 13.6.20111104 (HockeyApp 13.6 beta) on iPhone 6S iOS 13.2 with a Jetpack site `crooked-badger.jurassic.ninja` running WP 5.2.4 Jetpack 7.9.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
